File: wxdgaming-mmoarpg/cfg-client/code/Org/Wxd/Mmo/Core/Cfg/bean/mapping/QVipMapping.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace wxdgaming.mmo.Core.Cfg.Bean.Mapping
{

    /// <summary>
    /// excel 构建 Vip.xlsx - q_vip - vip等级
    /// </summary>
    public class QVipMapping
    {
        /// <summary>
        /// vip等级
        /// </summary>
        public int Id { get; set; }
        /// <summary>
        /// vip升级所需要的经验
        /// </summary>
        public int Exp { get; set; }
        /// <summary>
        /// vip名字
        /// </summary>
        public String VipName { get; set; }
        /// <summary>
        /// 解锁后获得奖励
        /// </summary>
        public int[][] Rewards { get; set; }
    }
}

File: wxdgaming-mmoarpg/wxdgaming.mmo.core/src/main/java/wxdgaming/mmo/core/cfg/bean/mapping/QVipMapping.java
package wxdgaming.mmo.core.cfg.bean.mapping;


import lombok.Getter;
import lombok.Setter;
import lombok.experimental.Accessors;
import wxdgaming.boot.batis.struct.DataChecked;
import wxdgaming.boot.batis.struct.DbColumn;
import wxdgaming.boot.batis.struct.DbTable;
import wxdgaming.boot.core.lang.ObjectBase;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;
import java.util.Set;


/**
 * excel 构建 Vip.xlsx - q_vip - vip等级
 **/
@Getter
@Setter
@Accessors(chain = true)
@DbTable(mappedSuperclass = true, name = "q_vip", comment = "Vip.xlsx - q_vip - vip等级")
public abstract class QVipMapping extends ObjectBase implements DataChecked, Serializable {

    /** vip等级 */
    @DbColumn(name = "id", key = true, comment = "vip等级")
    protected int id;
    /** vip升级所需要的经验 */
    @DbColumn(name = "exp", comment = "vip升级所需要的经验")
    protected int exp;
    /** vip名字 */
    @DbColumn(name = "vip_name", comment = "vip名字")
    protected String vipName;
    /** 解锁后获得奖励 */
    @DbColumn(name = "rewards", comment = "解锁后获得奖励")
    protected int[][] rewards;

}
